StreamingAnalytics/ResourceMonitoring/resource-monitor.py

```diff
@@ -0,0 +1,28 @@
+import psutil
+import paho.mqtt.client as mqtt
+import time
+
+mqttClient = mqtt.Client("Resource-monitor")
+broker = "localhost"
+
+def on_publish(client,userdata,result):             #create function for callback
+	print("data published \n")
+	pass
+
+def on_connect(client, userdate, flags, rc):
+	print("Connected to mosquitto with result {0}".format(rc))
```

Comment:
Might not be mosquitto - suggest replacing with:
print("Connected to MQTT broker on " + broker " + with result {0}".format(rc))
(or equivalent)

Reply:
Done